services/Authentication/src/Authentication.BusinessLogicLayer/AssemblyReference.cs

```diff
@@ -0,0 +1,3 @@
+namespace Authentication.BusinessLogicLayer;
+
+public class AssemblyReference;
```

Comment:
What do you need this class for?

Reply:
I am passing the AssemblyReference from the BLL as a parameter to the builder.Services.AddAutoMapper(typeof(AssemblyReference).Assembly) into the Presentation Layer for the sake of AutoMapper functioning